### PR TITLE
Bug redirect after signup

### PR DESCRIPTION
**Pull request by YumiKimbara**
After signing up and trying to go to the /signup page, it will redirect to the filter1 page.

### CODE REVIEW

File: src/routers/PublicRoute.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 export const PublicRoute = ({
-  isAuthenticated, 
+  isAuthenticated,
+  signedupAccount,
   component: Component,
   ...rest
 }) => (
   <Route {...rest} component={(props) => (
-    isAuthenticated ? (
+    isAuthenticated && Object.keys(signedupAccount).length !== 0 ? (
```
Comment (BlackStone8960) [inline src/routers/PublicRoute.js:13]: If signedupAccount is undefined, this conditional expression will occur error. Can you just consider about it as well?

**Review by BlackStone8960 — COMMENTED** [review]
Thanks, it worked correctly. Could you just check my comment? If it occurs error, application could be crashed.

File: src/routers/PublicRoute.js (hunk 1 above)
Comment (YumiKimbara) [reply src/routers/PublicRoute.js]: I fixed this conditional, but do you think it is okay...?
I don't know how to duplicate an error, so I tried to access the signup page multiple times after signing up, but I could not get any error.
Please let me know if there is a better solution for this!!

Comment (BlackStone8960) [reply src/routers/PublicRoute.js]: It looks good! I didn't get any error as well, but signedupAccount might be undefined for some reason, couldn't it? I thought it's important to consider about various patterns when implementing something, so I pointed out as I comment. If we use Typescript, such type of error can't happen so maybe we can use Typescript in the future, though. Anyway, thank you so much for doing good job!

**Review by BlackStone8960 — APPROVED** [review]

File: src/routers/PublicRoute.js (hunk 1 above)
Comment (YumiKimbara) [reply src/routers/PublicRoute.js]: I see! Thank you for checking and telling me the explanation :)

**Pull request merged**